frontend/src/components/Form/SimpleFormInput.tsx

```diff
@@ -1,6 +1,6 @@
 import { FieldValues, Path, UseFormRegister } from 'react-hook-form';
 
-import { HTMLInputTypeAttribute } from 'react';
+import { useState, HTMLInputTypeAttribute } from 'react';
```

Comment:
is `useState` needed?

Reply:
No, its not. I was initially going to use it, but I forgot to remove it in the end.

Reply:
Hm surprised this goes through the pre-commit hook. We have a linter setup that should execute when committing, I'll have to take a look at it.